### PR TITLE
Add wave background

### PR DESCRIPTION
**Pull request by alphakennyn**

### CODE REVIEW

**Comment by dannylekim** [comment]
Task linked: [CU-5ccwbj Login/Register page v2](https://app.clickup.com/t/5ccwbj)

**Comment by dannylekim** [comment]
Task linked: [CU-82dt3h Create background component: WaveBackground](https://app.clickup.com/t/82dt3h)

**Review by asianswagstorm — APPROVED** [review]

File: src/views/Background/__tests__/Bacground.test.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Background from '../Background.jsx';
+import { shallow } from "enzyme";
+
+describe('Background', () => {
+    describe("render", () => {
+        describe("snapshots 📸", () => {
+            it("BillDisplay should match snap shot if no p[rop provided", () => {
```
Comment (Kevinjhlin) [inline src/views/Background/__tests__/Bacground.test.jsx]: typo @ line 8 ;)

**Review by Kevinjhlin — APPROVED** [review]

**Review by briankimlim — APPROVED** [review]

**Comment by briankimlim** [comment]
remember when squash, to name commit in form:
`feat(#5ccwbj_#82dt3h): adds wave background`

**Pull request merged**